Source/WebCore/WebCore.xcodeproj/project.pbxproj

```diff
@@ -48002,13 +48002,27 @@
 			isa = XCBuildConfiguration;
 			baseConfigurationReference = 1CDD45E50BA9C84600F90147 /* WebCore.xcconfig */;
 			buildSettings = {
+				CODE_SIGN_IDENTITY = "Apple Development";
```

Comment:
this needs to be reverted. Don't push that.
There's no need to modify the project.pbxproj

Always check your PR after uploading or double-check the commit to make sure you don't get stray files included.